go.mod

```diff
@@ -7,3 +7,5 @@ require (
 	golang.org/x/crypto v0.10.0
 	golang.org/x/sys v0.9.0
 )
+
+replace github.com/google/go-pkcs11 v0.2.0 => github.com/lcforges/go-pkcs11 v0.0.0-20230808214450-8e4d0532171b
```

Comment:
Looks good but this change unmasked a regression in the CI.

Reply:
Ohh, you're right. I think this is the same issue I was having earlier with go test -v not passing but each test individually passing. I will look more into it